torchvision/ops/focal_loss.py

```diff
@@ -16,19 +16,20 @@ def sigmoid_focal_loss(
     Loss used in RetinaNet for dense detection: https://arxiv.org/abs/1708.02002.
 
     Args:
-        inputs: A float tensor of arbitrary shape.
+        inputs: (Tensor) A float tensor of arbitrary shape.
```

Comment:
The type is put before the colon character:
```suggestion
        inputs (Tensor): A float tensor of arbitrary shape.
```

The same everywhere else.

Reply:
Oh my bad . Sorry